app/models/concerns/system_like.rb

```diff
@@ -14,7 +14,7 @@ module SystemLike
 
   def compliant
     result = {}
-    profiles.map do |profile|
+    test_result_profiles.map do |profile|
```

Comment:
This is changed to keep the same logic for compliant, if I'm correct. :+1:

Reply:
Yes :)